kpi/views/browser_kpi_forecasts_2025.view.lkml

```diff
@@ -0,0 +1,155 @@
+view: browser_kpi_forecasts_2025 {
+  derived_table: {
+    sql:
+      WITH date_range AS (
+        SELECT DATE_ADD("2024-01-01", INTERVAL n DAY) AS submission_date
```

Comment:
there is also GENERATE_DATE_ARRAY. but maybe you need to do this for a reason im not seeing